src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -112,6 +113,9 @@ private static void runInspectAndDeidPipeline(
     if (options.getInputProviderType() == InputLocation.GCS && !usePubSub && !options.getProcessExistingFiles())
         throw new IllegalArgumentException("Either --processExistingFiles should be set to true or --gcsNotificationTopic should be provided");
 
+    if(options.getDataset()!= null && options.getOutputBucket()!=null)
```

Comment:
What does it take to generalise this to support file output instead of just GCS?

Reply:
I believe writing to file system is not possible for Dataflow runner